ultravox/model/ultravox_model.py

```diff
@@ -51,6 +51,10 @@ def __init__(self, config: UltravoxConfig):
         self.vocab_size = config.vocab_size
 
         self.audio_tower = self._create_audio_tower(config)
+        self.audio_tower_context_length = (
+            3000  # the context window for the whisper model
+        )
```

Comment:
We don't want Ultravox to only work with Whisper.

Reply:
Agreed, this length restriction should only apply to the whisper encoder